src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/StructureToPtrTests.cs

```diff
@@ -96,61 +47,65 @@ public void VerifyByValArrayInStruct()
                     new StructWithIntField { value = 5 }
                 }
             };
-            int size = Marshal.SizeOf(structure1);
+            int size = Marshal.SizeOf(structure);
             IntPtr memory = Marshal.AllocHGlobal(size);
             try
             {
-                Marshal.StructureToPtr(structure1, memory, false);
-                Marshal.StructureToPtr(structure1, memory, true);
+                Marshal.StructureToPtr(structure, memory, false);
+                Marshal.StructureToPtr(structure, memory, true);
             }
             finally
             {
                 Marshal.FreeHGlobal(memory);
```

Comment:
forgot to call DestroyStructure to release its field memory?

Reply:
Fixed